tests/integration/targets/iosxr_cliconf/tests/common/commit_conf.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- debug:
+    msg:
+      START iosxr_cliconf integration tests on connection={{ ansible_connection
+      }}
+
+- block:
```

Comment:
why is the block added here? if are adding it can we have a rescue section that clears configuration session?